iOS Example Frame/iOS Example Frame/MainViewController.swift

```diff
@@ -76,7 +70,7 @@ final class MainViewController: UIViewController, CardViewControllerDelegate, Th
         let paymentConfiguration = PaymentFormConfiguration(apiKey: "pk_test_6e40a700-d563-43cd-89d0-f9bb17d35e73",
                                                             environment: environment,
                                                             supportedSchemes: [.visa, .mastercard, .maestro],
-                                                            billingFormData: billingForm)
```

Comment:
I think, this needs to be filled with `billingForm`

Reply:
The design shows no pre filled billing form data. The test data is still there just not passed at this stage of the PR to match design layout.